src/server/api/routes/allUsers.router.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controller
+const allUsersController = require('../controllers/getAllUsers.controller');
+
+/**
+ * @swagger
+ * /allUsers:
+ *  get:
+ *    summary: Select all usres
```

Comment:
Not very super important but if you write "Select user for game Leaderboard" here or something which explains your endpoint. Because when all the endpoints will be merged yours will be lost so to recognize your component without going thru the code you need to change the description.